src/Symfony/Component/Form/Command/DebugCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Form\Console\Helper\DescriptorHelper;
+use Symfony\Component\Form\FormRegistryInterface;
+
+/**
+ * A console command for retrieving information about form types.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class DebugCommand extends Command
+{
+    private $formRegistry;
+    private $namespaces;
+
+    public function __construct(FormRegistryInterface $formRegistry, array $namespaces = array('Symfony\Component\Form\Extension\Core\Type'))
+    {
+        parent::__construct();
+
+        $this->formRegistry = $formRegistry;
+        $this->namespaces = $namespaces;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('debug:form')
```

Comment:
Could you update this accordingly to https://github.com/symfony/symfony/pull/23887 and add the defaultName ? Thanks :)

Reply:
Rebased & done.